tests/test_base_classifier.py

```diff
@@ -3,19 +3,15 @@
 from scr.base_classifier import BaseClassifier
 class TestBaseClassifier(unittest.TestCase):
     def setUp(self):
-        self.page = PageModel(page_units=[
-            PageModelUnit(id="phisical_model",
-                          sub_model=PhisicalModel(),
-                          extractors=[],
-                          converters={"words_model": WordsToOneBlock()})
-        ])
+        self.phisical_model = PhisicalModel()
 
-        self.page.read_from_file('../../PageR-main/tests/files/blocks.json')
+
+        self.phisical_model.read_from_file('files/blocks.json')
```

Comment:
change path tests/files/blocks.json (There is not import scr, because module is not install)